test/application.js

```diff
@@ -49,6 +49,27 @@ describe('Application', () => {
     it("sets application's JSONRPC", () => {
       expect(application.JSONRPC).to.be.an.instanceof(JSONRPC);
     });
+    
+    it("calls addEventListener", sinon.test(function() {
+      sinon.spy(document.body, 'addEventListener');
+      expect(document.body.addEventListener.calledOnce);
+    }));
+
+    describe('#imageRequestResize(event)', () => {
```

Comment:
this doesn't seem like it should be in the init describe, but its a bit hard to trrack all the blocks start/end on this PR